### PR TITLE
JAVA-25111: Adding missed modules

### PR DESCRIPTION
**Pull request by Bipinkumar27**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1220,6 +1223,11 @@
                 <module>gradle-modules/gradle/maven-to-gradle</module>
                 <module>persistence-modules/spring-data-neo4j</module>
                 <module>spring-actuator</module>
```
Comment (anastasiosioannidis) [inline pom.xml:1225]: Please remove spring-cloud-contract. 
This will be added by another ongoing jira.

Comment (Bipinkumar27) [reply pom.xml:1225]: done

**Comment by anastasiosioannidis** [comment]
retest this please

**Comment by anastasiosioannidis** [comment]
retest this please

**Pull request merged**